### ISSUE TITLE
Not numerically sorting

### ISSUE THREAD

**Issue by Rishat-F** [asottile/reorder_python_imports#251]
![wrong_order](https://user-images.githubusercontent.com/66554797/173251257-9c172fe9-1f93-439e-b63e-93dbd2a909c5.png)

**Comment by asottile** [comment]
correct, it is sorting lexicographically

in the future do not take screenshots of text -- they cannot be searched, copied, or viewed with my accessibility options

**Comment by Rishat-F** [comment]
Ok, got it. Thanks for answer!
